### PR TITLE
News site 

### PR DESCRIPTION
**Pull request by lisabergstrom**
Link to my devloyed project: https://lisa-travel-news.netlify.app/

### CODE REVIEW

File: code/style.css [hunk 1]
```diff
@@ -1,4 +1,227 @@
...
+ } 
+  
+   
+  /* tablet*/ 
+ 
+  @media (min-width: 668px) { 
+  
+header { 
+  justify-content: center; 
+}
```
Comment (jessand77) [inline code/style.css:159]: I suggest that more indentation is used here and for the rest of the code inside the media query selectors.

File: code/style.css [hunk 2]
```diff
@@ -1,4 +1,227 @@
...
+ } 
+
+nav a {
+   text-decoration: none;
+   padding: 20px;
+}
+
+.polaroid {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+   width: 280x
```
Comment (jessand77) [inline code/style.css:30]: I think you can remove this line if you want.

File: code/style.css [hunk 3]
```diff
@@ -1,4 +1,227 @@
...
+  
+header { 
+  justify-content: center; 
+} 
+  
+ nav { 
+  width: 60%;  
+  display: flex;  
+  justify-content: space-between; 
+}
```
Comment (jessand77) [inline code/style.css:165]: The nav items and the logo overlap a little bit when the screen width is somewhere between 668 and 788 px. Try width: 75% instead; at least it seemed to work when I did some trial and error in DevTools.

**Review by jessand77 — COMMENTED** [review]
The site looks good, the code is easy to understand and generally it has a good structure. The project fulfills the minimum requirements as well as the requirements of red and black level. Good work!

File: code/index.html [hunk 4]
```diff
@@ -1,22 +1,130 @@
...
-  </body>
-</html>
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Travel News</title>
+  <link rel="stylesheet" href="./style.css" />
+</head>
+
+<style>A {text-decoration: none;} </style>
```
[the remark this answers is not in the document]
Comment (lisabergstrom) [reply code/index.html:10]: I didn´t manage to write the code in the css, that´s why I put the code in the html 😄

File: code/index.html [hunk 5]
```diff
@@ -1,22 +1,130 @@
...
+  <!-- Logo and navbar -->
+
+  <header>
+    <img class="logo" src="Logo1.jpeg" alt="logo">
+    <h1 class="headertext">Travel News</h1>
+    <nav>
+      <ul>
+        <p class="hover-underline-animation"><a href="https://www.thesitemagazine.com/thesitemagazine">ABOUT</a></p>
+        <p class="hover-underline-animation"><a href="https://www.thesitemagazine.com/thesitemagazine">READ</a></p>
+      </ul>
```
[the remark this answers is not in the document]
Comment (lisabergstrom) [reply code/index.html:27]: Thanks for input!

File: code/index.html [hunk 6]
```diff
@@ -1,22 +1,130 @@
...
+  <!-- The other news  -->
+
+  <div class="Newsbox2">
+
+    <div class="flex-child 3"> 
+      <a href="https://www.thesitemagazine.com/call-for-submissions">
+      <figure class="polaroid">
+        <img class="image" src="girls.webp" alt="girld">
+    <div class="date">Jan 15 2022</div>
+    <div class="footer-text"> Studio Gang's "Actionable Idealism"</div>
```
[the remark this answers is not in the document]
Comment (lisabergstrom) [reply code/index.html:61]: Aha okay, thanks!

File: code/index.html [hunk 7]
```diff
@@ -1,22 +1,130 @@
...
+        <p class="hover-underline-animation"><a href="https://www.thesitemagazine.com/thesitemagazine">WRITE</a></p>
+        <p class="hover-underline-animation"><a href="https://www.thesitemagazine.com/thesitemagazine">BUY</a></p>
+       <ul>
+      </ul>
+    </nav>
+  </header>
+
+  <!-- The big news  -->
+
+  <div class="Newsbox1">
```
[the remark this answers is not in the document]
Comment (lisabergstrom) [reply code/index.html:38]: That´s true! It´s probably a better choice.

File: code/style.css (hunk 3 above)
Comment (lisabergstrom) [reply code/style.css:165]: It seems like its working! Thanks!

File: code/style.css (hunk 2 above)
Comment (lisabergstrom) [reply code/style.css:30]: I tried, but it wasn´t possible. The frame dissapeard.

File: code/style.css [hunk 8]
```diff
@@ -1,4 +1,227 @@
...
+
+.Newsbox2{
+  display: flex;
+  flex-wrap: wrap;
+}
+
+}
+     
+.image1{ 
+ width: 80%;
```
[the remark this answers is not in the document]
Comment (lisabergstrom) [reply code/style.css:223]: I tried to remove it but the webpage became a mess  so I think I need to have it :)

**Pull request closed without merging**